db/migrate/20200115232918_add_user_id_to_kudos.rb

```diff
@@ -0,0 +1,32 @@
+class AddUserIdToKudos < ActiveRecord::Migration[5.1]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+=begin
+      pt-online-schema-change -uroot --alter \
+      "ADD COLUMN user_id INT, ADD INDEX index_kudos_on_user_id (user_id)" \
+      D=$DATABASE,t=kudos --chunk-size=10k --max-flow-ctl 0 \
+      --pause-file /tmp/pauseme --max-load Threads_running=25 \
+      --critical-load Threads_running=400 --set-vars innodb_lock_wait_timeout=2 \
+      --alter-foreign-keys-method=auto --execute
```

Comment:
missing a \ at the end

Reply:
@zz9pzza - These should be 2 separate commands that I just provided for copy-paste convenience: the multi-line pt-osc command that's run on the command line ("pt-online-schema-change...") and a sql command that's run on the mysql server ("insert into..."). Or at least, I'm assuming you usually run those commands separately like that (this is the first migration I've written for OTW)?

I can add extra documentation in this comment to make that clearer? Or is there something else that seems broken?